CRM/Utils/HttpClient.php

```diff
@@ -152,6 +163,45 @@ public function get($remoteFile) {
     return array(self::STATUS_OK, $data);
   }
 
+  private function curlExecWithRedirects($ch)
+  {
```

Comment:
our convention is to put a space instead of a newline in front of the opening curly brace